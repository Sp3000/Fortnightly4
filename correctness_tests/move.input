move 0 0 1 1
place 0 0 red
move 0 0 1 1
place 1 1 green
place 1 2 blue
connect 1 1 1 2
place 5 5 yellow
move 1 2 4 4
move 1 2 4 3
place 5 6 blue
move 5 5 -2 -2
place 5 6 purple
move 3 3 -2 -2
place 0 1 red
connect 0 1 1 1
move 1 2 3 2
move 1 2 3 1
move 9 9 5 5
halt