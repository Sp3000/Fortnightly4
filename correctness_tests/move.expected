Error: move 0 0 1 1
Error: place 1 1 green
Error: move 1 2 4 4
Error: move 1 2 4 3
Error: place 5 6 purple
Error: move 3 3 -2 -2
Error: move 1 2 3 2
Error: move 9 9 5 5